src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/OrderMadeDefendantNotificationHandler.java

```diff
@@ -109,7 +113,12 @@ && hasTrackChanged(caseData)) {
         if (isSDODrawnPreCPRelease()) {
             return SCENARIO_AAA6_DEFENDANT_SDO_DRAWN_PRE_CASE_PROGRESSION.getScenario();
         }
+
+        if (isFinalOrderIssued(callbackParams)) {
+            return SCENARIO_AAA6_UPDATE_DASHBOARD_DEFENDANT_TASK_LIST_UPLOAD_DOCUMENTS_FINAL_ORDERS.getScenario();
```

Comment:
This new scenario should still trigger same notification as SCENARIO_AAA6_CP_ORDER_MADE_DEFENDANT, could you add that to the SQL file? You'll just need to put in the name of the preexisting notification